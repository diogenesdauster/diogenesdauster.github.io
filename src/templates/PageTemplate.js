import React from 'react'
import { graphql } from 'gatsby'
import Layout from '../components/layout/layout'
import Page from '../components/page/page'

const PageTemplate = ({ data }) => {
  const post = data.markdownRemark
  console.log(data)
  return (
    <Layout>
      <Page title={post.frontmatter.title}>
        <div dangerouslySetInnerHTML={{ __html: post.html }} />
      </Page>
    </Layout>
  )
}

export const query = graphql`
  query($slug: String!) {
    markdownRemark(fields: { slug: { eq: $slug } }) {
      html
      frontmatter {
        title
      }
    }
  }
`

export default PageTemplate
