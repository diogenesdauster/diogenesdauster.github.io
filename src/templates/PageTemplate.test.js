import React from 'react'
import renderer from 'react-test-renderer'
import PageTemplate from './PageTemplate'

/* eslint-disable */
describe('PageTemplate', () =>
  it('renders correctly', () => {
    const tree = renderer
      .create(<PageTemplate data={pageTemplateData} layoutData={sideData} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  }))
/* eslint-disable */
