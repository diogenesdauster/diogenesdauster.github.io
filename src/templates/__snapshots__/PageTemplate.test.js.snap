// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTemplate renders correctly 1`] = `
Array [
  <div
    className="sidebar"
  >
    <div
      className="container sidebar-sticky"
    >
      <div
        className="sidebar-about"
      >
        <h1>
          <a
            href="/"
          >
            <span
              aria-label="emoji"
              role="img"
            >
              👨🏻‍💻
               
            </span>
            <span>
              &lt;/
            </span>
            Diógenes Dauster
            <span>
              &gt;
            </span>
          </a>
        </h1>
        <p
          className="lead"
        >
          Code Has No Accent
        </p>
      </div>
      <nav
        className="sidebar-nav"
      >
        <Link
          className="sidebar-nav-item"
          to="/"
        >
          Home
        </Link>
        <Link
          className="sidebar-nav-item"
          to="about"
        >
          About
        </Link>
        <Link
          className="sidebar-nav-item"
          to="archive"
        >
          Archive
        </Link>
        <Link
          className="sidebar-nav-item"
          to="projects"
        >
          Projects
        </Link>
      </nav>
      <p>
        © 
        2018
        . All rights reserved.
      </p>
    </div>
  </div>,
  <div
    className="content container"
  >
    <div
      className="page"
    >
      <h1
        className="page-title"
      >
        About
      </h1>
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p>Software Developer very fascinated about software architecture. I'm a person who likes hardworking for solving issues and always is learn something new. I have knowledge about FullStack Development, Mobile Development and Totvs ERP. Recently a took a Mobile Post-Graduation and a nanodegree in the Udacity to deep my knowledge. I started my developer career when I was a teenager and since then I've been learning how can I be a better developer. Making simple code for the others, taking care about what I write to avoid issue and doing test as many as possible.</p>
<h1>Contact</h1>
<ul>
<li><a href=\\"mailto:diogenes.dauster@gmail.com\\">diogenes.dauster@gmail.com</a></li>
<li><a href=\\"https://github.com/diogenesdauster\\">github.com/diogenesdauster</a></li>
<li><a href=\\"https://www.linkedin.com/in/diogenesdauster/\\">linkedin.com/in/diogenesdauster</a></li>
</ul>",
          }
        }
      />
    </div>
  </div>,
]
`;
